src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3,38 +3,38 @@
 
 #include "pch.h"
 #include "TerminalPage.h"
-#include "Utils.h"
-#include "../../types/inc/utils.hpp"
+#include "TerminalPage.g.cpp"
+#include "RenameWindowRequestedArgs.g.cpp"
 
 #include <filesystem>
-#include <LibraryResources.h>
 
-#include "TerminalPage.g.cpp"
-#include <winrt/Windows.Storage.h>
+#include <inc/WindowingBehavior.h>
+#include <LibraryResources.h>
+#include <TerminalCore/ControlKeyStates.hpp>
+#include <til/latch.h>
 
-#include "TabRowControl.h"
+#include "../../types/inc/utils.hpp"
 #include "ColorHelper.h"
 #include "DebugTapConnection.h"
 #include "SettingsTab.h"
-#include "RenameWindowRequestedArgs.g.cpp"
-#include "../inc/WindowingBehavior.h"
-
-#include <til/latch.h>
+#include "TabRowControl.h"
 
 using namespace winrt;
+using namespace winrt::Microsoft::Terminal::Control;
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+using namespace winrt::Microsoft::Terminal::TerminalConnection;
+using namespace winrt::Microsoft::Terminal;
```

Comment:
@lhecker can you create a version of this PR that applies cleanly to 1.11 and 1.12? This sort of stuff sure doesn't make it easy :smile: